releases/release-1.31/release-notes/maps/pr-124017-map.yaml

```
@@ -0,0 +1,4 @@
+pr: 124017
+releasenote:
+  text: 'kube-controller-manager removes deprecated command flags: --volume-host-cidr-denylist
```

```suggestion
  text: 'Removed deprecated command flags --volume-host-cidr-denylist
    and --volume-host-allow-local-loopback from kube-controller-manager.'
```

https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653191306